### DISCUSSION TITLE
Azure AD logins with NocoDB [Dec 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4777] [Discord Threads]
> 🙋‍♂️ **beata**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### SSO is planned as part of enterprise edition. We havent started work on that yet. We're working to make the current product robust. Which SQL database are you using -  (SQLServer?)  
#  
> 🙋‍♂️ **beata**  
#### Yes, We've used SqlServer. Looking forward to the enterprise edition. Would there be a way around it, perhaps using noco JWT variables and/or Noco Auth API in the meantime?We're trying this as a pilot now to get the business approval for the broader use of NocoDB.  
#  
> 🙋‍♂️ **vollerempfang**  
#### <@310157751395287040> Hey there, we would also love to support the project with an enterprise plan. SSO (Azure) is also a must for us. Thanks in advance!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972074991246016522)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by vkoussouros-aitenders** [reply]
Hello @o1lab, is there a roadmap with estimated release date for enterprise edition including SSO ?
